content/post/2018-06-05_badcomkillsgoodanalytics.md

```diff
@@ -0,0 +1,88 @@
+
+## Talk about your work
+I gave a co-worker horrible advice about a presentation as he was preparing to show the Director of Marketing our research. I hadn’t worked much with this stakeholder, but remembered being told he really likes digging into the data. The presentation looked like a Stats 101 lecture. It had everything: coefficient significance tests, ROC curve plots, and the confusion matrix (which did just that). It was bad.
+
+He wanted to know what the statistical analysis meant in prescriptive terms. He had decisions to make on spending and targeting and this meeting was supposed to give him the details he needed. He was data-driven, he wanted to make decisions backed by solid research. However, we needed to frame the research in actions he could take.
+
+#### Improving “talk about your work”
+
+###### Scout First
+Do everything you can to avoid a meeting where you don’t know what the stakeholders expect from you. Maybe engineering wants to dig into the architecture details of deploying a model, or a non-technical manager wants to understand survey analysis. A quick 5-minute sit-down a few days before the “real” meeting goes a long way to making sure everyone is on the same page when the meeting happens.
+
+###### Trim the Fat
+We work hard to accomplish interesting things, and we want to share that with the rest of the business. But, it’s almost always too much content. Most human brains can track 3-5 things at the same time, so each “interesting” thing you add pushes your luck that the boss will forget the key finding or proposal that all the work went into. Give it room to breathe.
+
+## Talk about where the data can is buried
+Analytics’ ideas run on data. Without it they fizzle out with an exasperated “but we don’t have data on ____”. Data dictionaries and other aspirational processes are great, but in reality nuggets of data and business logic live all around the company. I recall being able to help one business unit solve a question by weaving together scattered pieces we’d heard about from three other departments.
+
+It might take a clever trick to tie it back to the rest of your data, but you’ll breathe new life into ideas and add new value to the company. With each discovery, the team gains new insights for those “what do we know” conversations in the company and unlocks ways to solve questions team members had considered impossible to answer.
+
+#### Improving “talk about where the data is buried”
+
+###### Follow Crumbs
+If a co-worker presents a data point that you’re unfamiliar with, or you think it came from some unfamiliar source, follow up and understand their process to get data. For instance, you might have third party tools and internal tools that measure the same metric, but those third party tools might also include some domain-specific data.
```

Comment:
Maybe change the last word in this paragraph from "data" to "assumptions" (if that captures your meaning)?

Reply:
It doesn't. I'm thinking about tools like GA which records transactions and other metrics that overlap with your own transaction DB, but GA also has a lot more attributes that a company might not be recording elsewhere.

I've also seen different departments have data partnerships with marketing data companies and inventory planning companies which show some of the same metrics the company's transactional DB has, but each system includes domain specific things, like forecast est or attribution of the transaction

Reply:
will slack about this

Reply:
edited after slack convo